src/Components/Nav/TopNav.jsx

```diff
@@ -9,7 +9,7 @@ import PersonIcon from "@material-ui/icons/Person";
 
 import { signOut } from "../../redux/Actions/login.js";
 
-import LogoRound from "../../images/logo-round.svg";
+import LogoRound from "../../images/logo/logo-round.svg";
```

Comment:
shouldn't we name this `transparent-round.svg` and `transparent-react.svg` for consistency?

Reply:
good point